src/post_transform.c

```diff
@@ -262,32 +260,263 @@ int pluto_intra_tile_optimize_band(Band *band, int num_tiled_levels,
         printf("[pluto-intra-tile-opt] Score for loop %d: %d\n", l, score));
     IF_DEBUG(pluto_loop_print(loops[l]));
   }
+  return best_loop;
+}
+
+/// For each statement in the input outermost band, the routine returns a per
+/// statement band. The returned band->loop will have a single statement. This
+/// is used to find statements in the input band that are distributed in the
+/// tile space.
```

Comment:
"distributed in the tile space" isn't clear. Did you intend to say that tile space loops fused and intra-tile loops distributed?